### ISSUE TITLE
File - scriptedamigaemulator.js

### ISSUE THREAD

**Issue by dosboxonline** [db48x/emularity#18]
Help download scriptedamigaemulator.js
Found this files on archive.org:
scriptedamigaemulator.js.gz - unzip but does not work
scriptedamigaemulator.js.map
scriptedamigaemulator.json 
-------------
Windows user

**Comment by pengan1987** [comment]
Yes, I find the same problem, I build this one from the original SAE repo, it should work
http://dnbwg.cdn.bcebos.com/emularity-common/emulators/sae/sae-merge.js
